<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScoreBoard</class>
 <widget class="QDialog" name="ScoreBoard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="img.qrc">
    <normaloff>:/image1/img/backgorund.jfif</normaloff>:/image1/img/backgorund.jfif</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">border-image: url(:/image1/img/scoreboard2.jfif);</string>
  </property>
  <widget class="QLabel" name="firstLabel">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>30</y>
     <width>191</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image1/img/whitebackground.jpeg);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#3f81a3;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="thirdLabel">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>130</y>
     <width>191</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image1/img/whitebackground.jpeg);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#3f81a3;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="secondLabel">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>100</y>
     <width>191</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image1/img/whitebackground.jpeg);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backScoreButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>620</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/loginscr/loginscreenimg/back_white.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
